Controller/DynamicController.php

```diff
@@ -93,11 +152,11 @@ public function cgetFieldsAction(Request $request)
 
         if ($form) {
             $locale = $this->getLocale($request);
-            $fieldDescriptors = $this->get('sulu_form.list_builder.dynamic_list_factory')
+            $fieldDescriptors = $this->dynamicListFactory
                 ->getFieldDescriptors($form, $locale);
         }
 
-        return $this->handleView($this->view(array_values($fieldDescriptors)));
+        return $this->viewHandler->handle($this->view(array_values($fieldDescriptors)));
     }
```

Comment:
I think you don't need this function at all anymore, so it can be safely deleted. This was the way the metadata for lists was loaded in the old UI.